src/Pages/suyeonkim/Main/Feeds/Feeds.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import Feed from './Feed/Feed';
+import './Feeds.scss';
+
+class Feeds extends React.Component {
+  constructor() {
+    super();
+
+    this.state = {
+      feeds: [],
+    };
+  }
+
+  componentDidMount() {
+    fetch('http://localhost:3000/data/suyeonkim/feed.json', {
+      method: 'GET',
+    })
+      .then(res => res.json())
+      .then(feedInfo => {
+        this.setState({
+          feeds: feedInfo,
+        });
+      });
+  }
+  render() {
+    const { feeds } = this.state;
+
+    const list = feeds.map(feedData => (
+      <Feed feedInfo={feedData} key={feedData.feedId} />
+    ));
+
+    return <ul className="feeds-page">{list}</ul>;
```

Comment:
```suggestion
    const list = ;

    return (
      <ul className="feeds-page">
        {feeds.map(feedData => (
          <Feed feedInfo={feedData} key={feedData.feedId} />
        ))}
      </ul>
    );
```
이 경우에는 jsx 자리에 합쳐 써주는 것이 어떨까요?